dockerfiles/theia-dev/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN npm config set prefix "${HOME}/.npm-global" && \
     echo "--global-folder \"${HOME}/.yarn-global\"" > ${HOME}/.yarnrc && \
     # add eclipse che theia generator
     yarn ${YARN_FLAGS} global add yo @theia/generator-plugin@0.0.1-1562578105 file:${HOME}/eclipse-che-theia-generator.tgz && \
+    echo "Installed npm Packages" && npm ls -g | sort | uniq && yarn global list && echo "End Of Installed npm Packages" && \
```

Comment:
could you please move that to https://github.com/redhat-developer/codeready-workspaces-theia

Reply:
I looked for files to edit in crw-theia but these changes seemed to be only applicable to the upstream templates. Can you tell me which files in crw-theia to update?